unskript-ctl/unskript_ctl_notification.py

```diff
@@ -406,7 +405,8 @@ def prepare_combined_email(self,
                                         parent_folder=parent_folder) is False:
                     self.logger.error("ERROR Archiving attachments")
                     raise ValueError("ERROR: Archiving attachments failed!")
-                target_file_name = tar_file_name
+                # Create temp tar file in accessible directory
+                target_file_name = os.path.join('/tmp', tar_file_name)
```

Comment:
`target_file_name = os.path.join("/tmp/, os.path.basename(self.execution_dir)) + ".tar.gz"`

This should have been outside the `if` condition block.